### DISCUSSION TITLE
Post api call returning "This joke was formatted incorrectly." respons

### DISCUSSION THREAD

**Discussion by vuk12** [Sv443/JokeAPI#258] [Q&A] [answered]
![4IyuRqVaZN](https://user-images.githubusercontent.com/37949896/112022094-3cd2b680-8b32-11eb-9ae0-d2b858ed1bd8.png)
I have followed instruction for API calls on https://v2.jokeapi.dev/#submit-endpoint, and use postman, but this is the message I got"This joke was formatted incorrectly.".
Also I have only URL and json Body that I copied from the link.

**Comment by Sv443** [comment, marked as answer]
You are sending the data encoded as `form-data`, which is not supported by the API. Please choose `raw`, select `json` in the dropdown that appears to the right and follow the format of the **example payload** shown in the [submit endpoint section of the docs.](https://jokeapi.dev/#submit-endpoint)  
Also please do me a favor and add the [`dry-run` parameter](https://jokeapi.dev/#dryrun-param) to the URL so you don't spam the joke submissions.

**Reply by vuk12** [reply]
It is working after changing data to raw. Thanks.
